generator/ServiceClientGeneratorLib/Generators/SourceFiles/EventStreamOutputGenerator.tt

```diff
@@ -104,6 +110,10 @@
             { "<#=member.PropertyName#>", payload => new <#=this.Config.ClassName#>EventStreamException(Encoding.UTF8.GetString(payload.Payload), new <#=member.ModelShape#>Unmarshaller().Unmarshall(EventStreamUtils.ConvertMessageTo<#=protocol#>Context(payload))) },
 <#
                 }
+                else if (protocol == "Cbor")
+                {
+                    // TODO
```

Comment:
Make the generator fail if we go in here to avoid us accidently shipping a service update and thinking it was successful